docs/4.4. Behaviour - Access Tokens.md

```diff
@@ -132,8 +132,8 @@ specifiers.
 A '`*`' wildcard can be used to replace zero or more path characters, and may encompass multiple path segments (e.g.
 `/single/*` and `/single/senders/*/constraints` would both correspond to
 `/single/senders/ea388089-9ffb-4a81-b109-a19da845b3b6/constraints`). It is RECOMMENDED that wildcards are used in
-place of URL parameters (such as specifying resource UUID's) to minimise token size. Specific URL parameters MAY be
-used in path specifiers if defining access to a small number of resources (such as a single node or device.)
+place of resource identifiers, such as specifying resource UUIDs, to minimise token size. Specific identifiers MAY
```

Comment:
Are you using 'resource' here in the general sense of "an API resource", or in the sense of "an NMOS Resource"?
Can probably delete "specifying resource" in any case, and then I think it reads OK?

Reply:
was intended to be NMOS resources